README.md

```diff
@@ -17,7 +17,7 @@ Amazon gift card among the respondents of our KubeOne Survey.
 
 ## Getting Started
 
-All user documentation is available at the
+All user documentation for the latest stable version (v1.0) is available at the
```

Comment:
We should drop `(v1.0)`, so we don't have to update it each time we release a new minor release.
```suggestion
All user documentation for the latest stable version is available at the
```